en/asset-workflow/prefab.md

```diff
@@ -1,79 +1,91 @@
+Prefabs are pre-configured game objects that can be used as templates when we dynamically generate nodes.
 
-## Creating a prefab
+## Creating Prefab
 
-After editing the Node in the scene, drag it directly from **Node Tree** to **Assets**:
+After editing the node in the scene, drag the node directly from the **Node Tree** to the **Assets** panel at:
 
 ![prefab](prefab/create.png)
 
-And thus you create a prefab:
+to create a **prefab**:
 
 ![prefab](prefab/created.png)
 
-## Saving prefab
+## Editing Prefab
 
-After changing the prefab object in the scene, click **Save** in the **Properties** panel and you can save the corresponding prefab:
+Double-clicking on a prefab in the **Assets** panel switches the scene editing mode of the **Scene** panel to prefab editing mode.
 
-![apply](prefab/apply.png)
+After editing, click the **Save** button on the top of the **Scene** panel to save the edited prefabs, and then click the **Close** button to exit prefab editing mode and return to scene editing mode.
 
-## Reverting prefab
+## Using Prefab
 
-After changing the prefab object in the scene, click **Go Back** in the **Properties** panel and you can restore the corresponding prefab to the status in **Assets**:
+Drag and drop a prefab from the **Assets** panel to the **Node Tree** or the **Scene** panel to generate a **Prefab Instance** in the scene. The Prefab Instance node is shown in blue in the **Node Tree**.
 
-![revert](prefab/revert.png)
+If you modify each property of a prefab instance in the scene and then save the scene, the modified data will be stored in that prefab instance and will not affect the prefabs in the **Assets** panel or the data of other prefab instances generated using prefabs.
 
-## Auto Sync and Manual Sync
+## Prefab Instances
 
-You can choose Auto Sync or Manual Sync for every prefab instances in the scene.<br>
-If set to **Manual Sync**, when its originating asset changed, the prefab instances will NOT dynamically refresh to stay synchronized with the originating asset. Refreshing only triggered when users manually revert the prefab.<br>
-If set to **Auto Sync**, the prefab instances will dynamically refresh to stay synchronized with the originating asset.
+When the prefab instance node is selected in the **Node Tree**/**Scene** panel, several actionable buttons appear at the top of the **Properties** panel panel.
 
-The icon represents current sync mode for the selected prefab, you can switch between two modes by clicking the icon:
+![edit in scene](prefab/edit-in-scene.png)
 
-![non-syncable](prefab/non-syncable.png)
+- **Select**: click this button to yellow highlight the prefab for this prefab instance in the **Assets** panel.
 
-The icon above represents the currently selected prefab uses Manual Sync mode, click the icon will switch to Auto Sync mode:
+- **Return**: if the prefab instance is modified, click the **Return** button to restore the prefab instance to the initial state.
```

Comment:
```suggestion
- **Go back**: if the prefab instance is modified, click the **Return** button to restore the prefab instance to the initial state.
```

Reply:
either is fine.

Reply:
The editor is written with **Go back**. The document needs to be consistent with the editor.